ftw/referencewidget/resources/refwidget.js

```diff
@@ -0,0 +1,167 @@
+$(function() {
+    $(document).on('click', '.referencewidget button', openOverlay);
+
+    $(document).on('click', '.refbrowser .path span', jump_to);
+    $(document).on('click', '.refbrowser .search button', search);
+    $(document).on('keypress', '.refbrowser .search input', function(event){
+        if(event.which == 13) {search(event);}});
+
+    $(document).on('change', '.refbrowser .listing input.ref-checkbox', checkbox_flipped);
+    $(document).on('click', '.refbrowser button.cancel', function(){$('.refbrowser').remove()});
+    $(document).on('click', '.refbrowser .ref_list_entry', switch_level);
+    $(document).on('click', '.refbrowser .listing input.ref-checkbox', function(e) {e.stopPropagation();});
+    request_data = {}
+    url = location.protocol + '//' + location.host + location.pathname;
+    widget_url = ""
```

Comment:
@Tschanzt I would recommend using [eslint](http://eslint.org/ ). And [SublimeLinter-eslint](https://github.com/roadhump/SublimeLinter-eslint)
Here is my .eslintrc:
```json
{
  "extends": "eslint:recommended",
  "env" : {
    "node" : true,
    "amd" : true,
    "jquery" : true,
    "mocha" : true,
    "phantomjs" : true,
    "browser" : true
  },
  "rules" : {
    "quotes": [1, "double", "avoid-escape"],
    "no-underscore-dangle": [0],
    "semi": ["error", "always"]
  },
  "parserOptions": {
      "sourceType": "script",
      "ecmaVersion": 5
  }
}
```
Store is under `~/.eslintrc` to have a global config.